agent-inject/agent/container_sidecar.go

```diff
@@ -15,7 +15,7 @@ const (
 	DefaultResourceLimitMem   = "128Mi"
 	DefaultResourceRequestCPU = "250m"
 	DefaultResourceRequestMem = "64Mi"
-	DefaultContainerArg       = "echo ${VAULT_CONFIG?} | base64 -d > /tmp/config.json && vault agent -config=/tmp/config.json"
+	DefaultContainerArg       = "echo ${VAULT_CONFIG?} | base64 -d > /home/vault/config.json && vault agent -config=/home/vault/config.json"
```

Comment:
Does this have any implications if a user already has Vault setup with a prior version of the helm chart, and then they upgrade the helm chart? Will the `/home` dir exist?

Reply:
It exists already and when they upgrade it will reinject the the containers with the new configuration.